assets/js/modules/ads/pax/services.test.js

```diff
@@ -132,36 +132,20 @@ describe( 'PAX partner services', () => {
 
 				it( 'should hold correct value for conversionLabels property when data is present', async () => {
 					const mockSupportedEvents = [ 'mock-event' ];
-					const adsModuleDataVar = '_googlesitekitModulesData';
-					const adsModuleDataVarValue = {
-						ads: {
-							supportedConversionEvents: mockSupportedEvents,
-						},
-					};
-
-					global[ adsModuleDataVar ] = adsModuleDataVarValue;
```

Comment:
Here the global wasn't necessary because this is being used internally by a selector so we should provide this as usual instead, and test the behavior of the resolver in the datastore tests.